docs/tutorials/fabric_remote.md

```diff
@@ -0,0 +1,425 @@
+    my-org:
+        certificateAuthorities:
+            - my-fabric-network-ca
+        cryptoPath: /etc/firefly/msp
+        mspid: my-org-mspid
+        peers:
+            - my-peer-node
+peers:
+    my-peer-node:
+        tlsCACerts:
+            path: /etc/firefly/msp/tlscacerts/tls-ca-cert.pem
+        url: grpcs://my-peer-node.my-domain.com:7051
+version: 1.1.0%
+
+```
+
+> **NOTE:** If you had multiple members in your stack, with each member representing a different fabric organizational entity, then you would need to add separate ccp.yml files for all of them in a similar manner.
+
+## Setting up Batchpin chaincode and Running your stack
+
+### Deploying batchpin chaincode to fabric network
+
+For Firefly to succeesfully integrate and interact with your remote fabric network, you need to also deploy a Firefly batchpin chaincode in your network setup. This chaincode can be found in the [FireFly git repo](https://github.com/hyperledger/firefly/tree/main/smart_contracts/fabric/firefly-go).
+
+Once you have successfully packaged, installed, approved and commited this chaincode in your remote fabric network, carefully note down the chaincode name you gave and the channel name where this chaincode is deployed for future reference.
+
+> **NOTE:** You can deploy the batchpin chaincode to the same channel where your application chaincode lives or to a separate channel as well.
+
+### Update batchpin chaincode details in firefly core config
+
+In `$HOME/.firefly/stacks/my-remote-fabric/init/config`, you will find a firefly_core_0.yml file which is used by firefly core service in our firefly network setup.
+
+In this file, under the fabconnect section of blockchain-fabric section, you need to change the chaincode value to your batchpin chaincode name, and the channel value to the channel name where your batchpin chaincode was deployed in the previous step.
+
+### Start your stack
+
+To start your stack now, simply run:
+
+```
+ff start my-remote-fabric
+```
+
+This may take a minute or two and in the background the FireFly CLI will do the following for you:
+
+- Download Docker images for all of the components of the Supernode
+- Set up configuration between all the components
+- Deploy FireFly's `BatchPin` smart contract to our remote fabric network
+- Register an identity for each member and node in our remote fabric network
+- It will probably also initialize a new blockchain and blockchain node running inside a container and also deploy an `ERC-1155` token smart contract to that local fabric network. This behaviour can be ignored.
+
+After your stack finishes starting it will print out the links to each member's UI and the Sandbox for that node. For this guide, it's just one member.
+
+```
+Web UI for member '0': http://127.0.0.1:5000/ui
+Sandbox UI for member '0': http://127.0.0.1:5108
+
+To see logs for your stack run:
+
+ff logs my-remote-fabric
+
+```
+
+## Define and Broadcast FireFly Interface Document for your chaincode
```

Comment:
It looks like most of this section and the rest of this guide are copied from this page: https://hyperledger.github.io/firefly/tutorials/custom_contracts/fabric.html

I think we can have a short description of what you need to do next here, but then link to that page for the details. That way we only have one page we need to update / add to when we need to talk about using custom contracts with Fabric networks.

If you feel there is information missing from that tutorial (I see you added some detail about the Sandbox here which is great!) please feel free to update the other tutorial as well.

Reply:
Addressed.